src/util-base64.c

```diff
@@ -387,7 +387,7 @@ static int B64DecodeStringEndingSpaces(void)
     uint32_t consumed_bytes = 0, num_decoded = 0;
     uint8_t dst[10];
     Base64Ecode code = DecodeBase64(dst, sizeof(dst), (const uint8_t *)src, 9, &consumed_bytes,
-            &num_decoded, BASE64_MODE_RFC2045);
+            &num_decoded, Base64ModeRFC2045);
```

Comment:
If you wanted to keep the all caps you can suppress the warning with `allow(non_camel_case_types)`.  Check the top of `core.rs` for an example.

But this is looking good.